### ISSUE TITLE
Tool for generating dynamic client code from GraphQL queries

### ISSUE THREAD

**Issue by jmartisk** [smallrye/smallrye-graphql#1029]
Perhaps it would be useful to have a script that consumes a GraphQL query and transforms that into a Java snippet for using dynamic client. Writing queries for the dynamic client  can get quite cumbersome, this could alleviate some of the pain.

**Comment by t1** [comment]
There is an experimental generator for the typesafe client.

**Comment by jmartisk** [comment]
Yeah, just with this one I'd probably go really lightweight - it's supposed to generate just snippets, not self-contained Java classes, so it could be a script written in Python or something.
